src/web/package.json

```diff
@@ -16,12 +16,15 @@
     "axios": "^0.19.2",
     "bootstrap-vue": "^2.11.0",
     "core-js": "^3.6.5",
+    "dom-to-image": "^2.6.0",
     "eslint-config-prettier": "^6.10.1",
```

Comment:
we still using these other packages?

Reply:
I'm not too sure, but I only added that DOM-To-Image One.